lib/src/ui/common/util/date_time_formatter.dart

```diff
@@ -0,0 +1,5 @@
+String formatDateTimeToDateString(DateTime dateTime) =>
```

Comment:
a gdyby zrobić coś ala
```
abstract class DateTimeFormatter {
    formatDate(dateTime: DateTime)
    formatTime(dateTime: DateTime)
}
```
i wstrzykiwać implementację?

Ewenutalnie może extension na DateTime? wtedy byłoby coś typu `dateTime.formattedTime` i `dateTime.formattedDate`

Wstrzykiwanie dałoby możliwość zmiany implementacji bez zmiany interfejsu, a jak wolisz extension to jako, że język pozwala na extensiony to zawsze naturalniej wyglądają niż metody utilowe ;)

Reply:
zrobiłem extension